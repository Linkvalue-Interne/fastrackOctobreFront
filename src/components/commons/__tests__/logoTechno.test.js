import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-emotion';

import Img from '../logoTechno';

expect.addSnapshotSerializer(serializer);

describe('cardPartnerCSS', () => {
  it('renders with correct styles', () => {

    const tree = renderer.create(<Img />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
