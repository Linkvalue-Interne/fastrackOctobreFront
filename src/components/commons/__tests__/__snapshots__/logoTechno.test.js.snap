// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders with correct styles 1`] = `
.emotion-0 {
  height: 35%;
  width: 35%;
  margin-bottom: 1rem;
}

<img
  className="emotion-0"
/>
`;
